### PR TITLE
Fix minor issues in DARTS and add a simple CIFAR10 example (with random mutator)

### PR DESCRIPTION
**Pull request by ultmaster**
Will fix merging issues tomorrow.

### CODE REVIEW

**Comment by chicm-ms** [comment]
@xuehui @chicm-ms

File: src/sdk/pynni/nni/nas/pytorch/random/mutator.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import torch
```
Comment (xuehui1991) [inline src/sdk/pynni/nni/nas/pytorch/random/mutator.py]: license?

Comment (xuehui1991) [inline src/sdk/pynni/nni/nas/pytorch/random/mutator.py]: license

File: examples/nas/random/train.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        x = self.gap(x).view(bs, -1)
+        x = F.relu(self.fc1(x))
+        x = F.relu(self.fc2(x))
+        x = self.fc3(x)
+        return x
+
+
+if __name__ == "__main__":
+    transform = transforms.Compose(
+        [transforms.ToTensor(), transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))])
```
Comment (xuehui1991) [inline examples/nas/random/train.py]: transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))?

You could reference here:
https://github.com/kuangliu/pytorch-cifar/blob/master/main.py

File: examples/nas/random/train.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            loss.backward()
+            optimizer.step()
+
+            # print statistics
+            running_loss += loss.item()
+            if i % 100 == 99:    # print every 20 mini-batches
+                print('[%d, %5d] loss: %.3f' % (epoch + 1, i + 1, running_loss / 100))
+                running_loss = 0.0
+
+    print("Finished training. Final architecture:")
```
Comment (xuehui1991) [inline examples/nas/random/train.py]: could you provide some examples about  how to use this architecture to retrain?

File: examples/nas/random/train.py (hunk 2 above)
Comment (ultmaster) [reply examples/nas/random/train.py]: This is an example to demonstrate the power of our interface, not for obtaining the high accuracy. The code is copied from [here](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#sphx-glr-beginner-blitz-cifar10-tutorial-py). I think putting a notebook is better but that will be in `notebooks` folder?

File: examples/nas/random/train.py (hunk 3 above)
Comment (ultmaster) [reply examples/nas/random/train.py]: Example for retrain is already in darts. This is a simple example.

File: src/sdk/pynni/nni/nas/pytorch/random/mutator.py (hunk 1 above)
Comment (ultmaster) [reply src/sdk/pynni/nni/nas/pytorch/random/mutator.py]: License will be included in a PR after cutting branch, as discussed before.
